torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -133,36 +184,40 @@ TensorPipeAgent::~TensorPipeAgent() {
 
 void TensorPipeAgent::startImpl() {
   VLOG(1) << "RPC agent for " << workerInfo_.name_ << " is starting";
-  auto uvContext = std::make_shared<tensorpipe::transport::uv::Context>();
 
-  std::string uvAddress = guessAddress(*uvContext);
-  VLOG(1) << "RPC agent for " << workerInfo_.name_ << " is using address "
-          << uvAddress;
+  std::vector<std::string> addresses;
+  int lowestPriority = -1;
```

Comment:
Would I be correct if I assume higher the number lower the priority?

Reply:
Very good catch, thanks! This was indeed the case, because of a long-standing bug of TensorPipe where, counterintuitively, lower priorities took precedence over higher ones. I ended up fixing that bug just before sending out this diff (https://github.com/pytorch/tensorpipe/commit/8fb1fe66f8fe7d95f4b3aa934afc9b194cd47db6) but forgot to update this constant.